import-service/handler.js

```diff
@@ -68,8 +70,26 @@ module.exports.parseProductsFile = async function(event) {
       };
 
       return new Promise((resolve, reject) => {
-        s3.upload(parsedParams, (err, data) => {
+        s3.upload(parsedParams, async (err, data) => {
```

Comment:
Why don't you use `s3.copyObject`? Won't it be faster for S3 to just copy what it has locally instead of transferring all the data across the network?